### ISSUE TITLE
Missing configuration in conf.yml of deepstream.io-mac-2.3.2.zip

### ISSUE THREAD

**Issue by honorcode** [deepstreamIO/deepstream.io#705]
outgoingBufferTimeout and storageHotPathPatterns configurations missing in conf.yml.

**Comment by honorcode** [comment]
Are the following MAC default config.yml diffs EXPECTED between config.yml files for v2.3.2 in MAC release package and the deepstream.io github repo conf/config.yml?

A diff of the downloaded zip conf/config.yml (mac 2.3.2) and the deepstream.io server conf/config.yml shows **these sections missing** from the release package config.yml:

# Connection Endpoint Configuration

```
# Connection Endpoint Configuration
# currently at most one may be specified
connectionEndpoints:
  websocket:
    name: uws
    options:
        # port for the websocket server
        port: 6020
        # host for the websocket server
        host: 0.0.0.0
        # url path websocket connections connect to
        urlPath: /deepstream
        # url path for http health-checks, GET requests to this path will return 200 if deepstream is alive
        healthCheckPath: /health-check
        # the amount of milliseconds between each ping/heartbeat message
        heartbeatInterval: 30000
        # the amount of milliseconds that writes to sockets are buffered
        outgoingBufferTimeout: 0

        # Security
        # amount of time a connection can remain open while not being logged in
        unauthenticatedClientTimeout: 180000
        # invalid login attempts before the connection is cut
        maxAuthAttempts: 3
        # if true, the logs will contain the cleartext username / password of invalid login attempts
        logInvalidAuthData: false
        # maximum allowed size of an individual message in bytes
        maxMessageSize: 1048576

...

# A list of prefixes that, when a record is updated via setData and it matches one of the prefixes
# it will be permissioned and written directly to the cache and storage layers
# storageHotPathPatterns:
  # - analytics/
  # - metrics/

```

And **this section INCLUDED** in release packager config.yml version **BUT NOT FOUND** in the server repo proj conf/config.yml for tag 2.3.2:

```
# Security
# amount of time a connection can remain open while not being logged in
unauthenticatedClientTimeout: 180000
# invalid login attempts before the connection is cut
maxAuthAttempts: 3
# if true, the logs will contain the cleartext username / password of invalid login attempts
logInvalidAuthData: true
# maximum allowed size of an individual message in bytes
maxMessageSize: 1048576
```

**Comment by yasserf** [comment]
Should be fixed in 2.3.3

Thanks for raising!
